.github/workflows/firefox-ios-ui-tests.yml

```diff
@@ -70,7 +71,29 @@ jobs:
         run: |
           gem install xcpretty -v 0.3.0
           pip install blockkit==1.9.1
-          npm install -g junit-report-merger@7.0.0       
+          npm install -g junit-report-merger@7.0.0
+      - name: Ping URLs
+        id: urls
+        run: |
+          curl https://www.firefox.com > /dev/null
+          curl https://www.pocket.com > /dev/null
+          curl https://wikipedia.org > /dev/null
+          curl https://www.walmart.com > /dev/null
+          curl https://jigsaw.w3.org/HTTP/Basic/ > /dev/null
+          curl https://storage.googleapis.com/mobile_test_assets/public/pdf-test.pdf > /dev/null
+          curl https://www.ebay.com > /dev/null
+          curl https://www.amazon.com > /dev/null
+          curl https://www.forbes.com > /dev/null
+          curl https://theguardian.com > /dev/null
+          curl https://www.reddit.com > /dev/null
+          curl http://www.iana.org/domains/example > /dev/null
+          curl https://www.twitter.com > /dev/null
+          curl https://www.youtube.com > /dev/null
+          curl https://www.instagram.com > /dev/null
+          curl https://www.euronews.com > /dev/null
+          curl https://www.fakespot.com > /dev/null
+          curl https://www.badssl.com > /dev/null
+          curl https://accounts.google.com > /dev/null
```

Comment:
If the URLs are available, I've no idea why the tests have trouble loading some of the websites intermittently.